pkg/elevator/internal/webhooks/tenantobj_webhook.go

```diff
@@ -98,82 +97,31 @@ func (r *TenantObjWebhookHandler) Handle(ctx context.Context, req admission.Requ
 			fmt.Errorf("DerivedCustomResource object is missing version expose config for version %q", version))
 	}
 	// prepare config
-	_, otherFields := elevatorutil.SplitStatusFields(exposeConfig.Fields)
+	_, nonStatusExposedFields := elevatorutil.SplitStatusFields(exposeConfig.Fields)
 
 	providerObj := &unstructured.Unstructured{}
 	providerObj.SetGroupVersionKind(r.ProviderGVK)
 	providerObj.SetName(obj.GetName())
 	providerObj.SetNamespace(obj.GetNamespace())
 
-	// Check if the ProviderObj has already been created, if it is created, then regards this request
```

Comment:
I see no reason why you have removed all the comments, even you are going to use `Patch`, the comments are still valid, you just need to add some more comments about why you use `Patch`.  I don't think anyone will remember why use `Patch` not `req.Operation` after 6 months.